### PR TITLE
Rahul

### PR DESCRIPTION
**Pull request by rranjan14**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  border-width: 10px 10px 10px 0;
+  border-color: transparent white transparent transparent;
+  }
+  .left::after, .right::after {
+  left: 15px;
+  }
+ .right {
+  left: 0%;
+  }
+}
```
Comment (abhieshekumar) [inline css/style.css:131]: You are not using this file. You may delete it I guess.

**Review by abhieshekumar — CHANGES_REQUESTED** [review]

**Comment by rranjan14** [comment]
Okay

File: css/style.css (hunk 1 above)
Comment (rranjan14) [reply css/style.css:131]: not using the whole file anyways

Comment (abhieshekumar) [reply css/style.css:131]: then delete those file/folder and do push it. it will automatically reflect on this pull request

**Pull request closed without merging**